docs/docusaurus.config.js

```diff
@@ -0,0 +1,87 @@
+module.exports = {
+  title: 'Chart.js',
+  tagline: 'Open source HTML5 Charts for your website',
+  url: 'https://chartjs.org',
+  baseUrl: '/',
+  favicon: 'img/favicon.ico',
+  organizationName: 'chartjs', // Usually your GitHub org/user name.
+  projectName: 'chartjs.github.io', // Usually your repo name.
+  plugins: ['@docusaurus/plugin-google-analytics'],
+  scripts: ['https://www.chartjs.org/dist/next/Chart.min.js'],
```

Comment:
This is not good for `latest`?

Reply:
I fixed it to have the deploy script update it to point at the right location